docs/docfx.json

```diff
@@ -72,7 +75,7 @@
         ],
 
         "postProcessors": [],
-        "markdownEngineName": "markdig",
+        "markdownEngineName": "dfm",
```

Comment:
Hmm, I really wish I had documented why I switched from `dfm` to markdig at this point. :grimacing: